src/main/java/com/snowflake/kafka/connector/internal/SnowflakeTelemetryServiceV1.java

```diff
@@ -229,20 +146,23 @@ private String getAppName()
     return name;
   }
 
+  private String getTaskID()
+  {
+    if (taskID == null || taskID.isEmpty())
+    {
+      logWarn("taskID in telemetry service is empty");
+      return "empty_taskID";
+    }
+    return taskID;
+  }
+
   private enum TelemetryType
   {
     KAFKA_START("kafka_start"),
     KAFKA_STOP("kafka_stop"),
     KAFKA_FATAL_ERROR("kafka_fatal_error"),
-    KAFKA_NONFATAL_ERROR("kafka_nonfatal_error"),
-    KAFKA_USAGE("kafka_usage"),
-    KAFKA_CREATE_TABLE("kafka_create_table"),
-    KAFKA_REUSE_TABLE("kafka_reuse_table"),
-    KAFKA_CREATE_STAGE("kafka_create_stage"),
-    KAFKA_REUSE_STAGE("kafka_reuse_stage"),
-    KAFKA_CREATE_PIPE("kafka_create_pipe"),
-    KAFKA_FILE_FAILED("kafka_file_failed"),
-    KAFKA_SNOWFLAKE_THROTTLE("kafka_snowflake_throttle");
+    KAFKA_PIPE_USAGE("kafka_pipe_usage"),
```

Comment:
Is this being used anywhere

Reply:
This is for uploading pipeStatus object, which is removed in this PR. I should have removed this in this PR as well.